src/mlpack/methods/cf/normalization/z_score_normalization.hpp

```diff
@@ -0,0 +1,162 @@
+/**
+ * @file z_score_normalization.hpp
+ * @author Wenhao Huang
+ *
+ * This class performs z-score normalization on raw ratings.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_CF_NORMALIZATION_Z_SCORE_NORMALIZATION_HPP
+#define MLPACK_METHODS_CF_NORMALIZATION_Z_SCORE_NORMALIZATION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace cf {
+
+/**
+ * This normalization class performs z-score normalization on raw ratings.
+ *
+ * An example of how to use ZScoreNormalization in CF is shown below:
+ *
+ * @code
+ * extern arma::mat data; // (user, item, rating) table
```

Comment:
Really picky comment, but do you mind to use complete sentences (https://github.com/mlpack/mlpack/wiki/DesignGuidelines#comments) on the example as well?